src/fsm2c.py

```diff
@@ -904,6 +920,7 @@ def gen_header(rules, groupby, out=sys.stdout):
     g.o("#define EXPIRES_NEVER %d" % EXPIRES_NEVER)
     g.o("#include <json-c/json.h>")
     g.o('#include "utils.h"')
+    g.o('#include "hyperloglog.h"')
```

Comment:
Same here, do we really need everything from `hyperloglog.h`? Could be just an opaque forward definition of `hyperloglog_t` + function declarations. Not having this saves compilation time at runtime (when .tr file  is compiled).